src/styles/style.css

```diff
@@ -1,4 +1,7 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
```

Comment:
@import is a bad way of importing fonts (bad performance). Consider using a google font link in the +layout.svelte.

Reply:
fixed this issue